style.css

```diff
@@ -0,0 +1,45 @@
+#main-doc {
+  display: flex;
+  flex-direction: row-reverse;
+  /* position: absolute; */
+}
+maain {
+  font-color: red;
+}
+.content1 {
+  padding-left: 50px;
+  font-size: 18px;
+}
+.content {
+  background: #f7f7f7;
+  padding: 5px;
+  border-radius: 5px;
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+#navbar {
+  border-right: 2px solid grey;
+  width: 50rem;
+  /* padding: 10px; */
```

Comment:
Kindly next time try to delete the commented/unused code. This helps in making your code base appear cleaner and much more organised.

Reply:
Good job here 👍